src/javascript/app_2/App/Components/Elements/Calendar/panels/calendar_days.jsx

```diff
@@ -37,13 +53,17 @@ const getDays = ({ calendar_date, date_format, max_date, min_date, start_date, o
             // for forward starting accounts, only show same day as start date and the day after
             (start_date && (moment_date.isBefore(moment_start_date) || moment_date.isAfter(moment_start_date.clone().add(1, 'day'))));
 
+        // show 'disabled' style for dates that is not in the same calendar month,
+        // but the date should still be clickable
+        const is_other_month = moment_date.month() !== moment_cur_date.month();
         days.push(
             <span
                 key={date}
-                className={classNames('calendar-date', {
-                    active  : is_active && !is_disabled,
-                    today   : is_today,
-                    disabled: is_disabled,
+                className={classNames('calendar__cell', {
+                    'calendar__cell--is-active'     : is_active && !is_disabled,
```

Comment:
I think it is better to use the modifier name, instead of its binary value. For example, `calendar__cell--active`, `calendar__cell--disabled`, etc. extra word is not needed, since `--mod` should be clear enough to understand the modifier.